openassessment/templates/openassessmentblock/grade/oa_grade_complete.html

```diff
@@ -1,337 +1,197 @@
-                                            </li>
-                                            {% endif %}
-                                        {% endfor %}
-                                    {% endwith %}
-                                {% endfor %}
-
-                                {% for part in self_assessment.parts %}
-                                    {% if part.option.criterion.name == criterion.name %}
-                                <li class="answer self-assessment"
-                                    id="question--{{ criterion_num }}__answer--self">
-                                    <h5 class="answer__title">
-                                        <span class="answer__source">
-                                          <span class="answer__source__value">{% trans "Your Self Assessment" %}</span>
-                                        </span>
-                                        <span class="answer__value">
-                                            <span class="answer__value__label sr">{% trans "Your Assessment" %}: </span>
-                                            <span class="answer__value__value">
-                                                {{ part.option.label }}
-
-                                                <span class="ui-hint hint--top" data-hint="{{ part.option.explanation }}">
-                                                  <i class="icon fa fa-info-circle"
-                                                     title="{% blocktrans with name=part.option.label %}More information about {{ name }}{% endblocktrans %}"></i>
+            </h2>
+        </header>
+
+        <div class="ui-toggle-visibility__content">
+            <div class="wrapper--step__content">
+                <div class="step__content">
+                    <article class="submission__answer__display step__content__section">
+                        <h3 class="submission__answer__display__title">{% trans "Your Response" %}</h3>
+
+                        {% include "openassessmentblock/oa_submission_answer.html" with answer=student_submission.answer answer_text_label="Your response to the question above:" %}
+
+                        {% include "openassessmentblock/oa_uploaded_file.html" with file_upload_type=file_upload_type file_url=file_url header="Your Upload" class_prefix="submission__answer" %}
+                    </article>
+
+                    <article class="submission__peer-evaluations step__content__section">
+                        <h3 class="submission__peer-evaluations__title">{% trans "Assessments of Your Response" %}</h3>
+
+                        <ol class="list submission__peer-evaluations__questions">
+                            {% for criterion in grade_details.criteria %}
+                                {% with criterion_num=forloop.counter %}
+                                    <li class="question question--{{ criterion_num }} ui-toggle-visibility">
```

Comment:
beyond scope of this PR, but anything with ui-toggle-visibility is not keyboard accessible and contents are not actually hidden in the DOM (i think they are given a zero height): https://openedx.atlassian.net/projects/AC/issues/AC-316